metals/src/main/scala/scala/meta/internal/builds/GradleDigest.scala

```diff
@@ -24,9 +26,10 @@ object GradleDigest extends Digestable {
   }
 
   def digestBuildSrc(path: AbsolutePath, digest: MessageDigest): Boolean = {
-    Files.walk(path.toNIO).iterator().asScala.forall { file =>
-      Digest.digestFile(AbsolutePath(file), digest)
+    WalkFiles.foreach(path) { file =>
```

Comment:
It would be nice to have the forall method in ListFiles, returning `true` doesn't look good.